tests/test_event.py

```diff
@@ -115,16 +123,10 @@ def test_api_get_all_user_events(self):
 
     def test_api_get_user_single_event(self):
         """Test API can get a single event by using it's id."""
-        self.register_user()
-        result = self.login_user()
-        access_token = json.loads(result.data.decode())['access_token']
+        access_token = self.get_access_token()
+        event_creation = self.event_creation()
 
-        rv = self.client().post(
-            '/api/events',
-            headers=dict(Authorization="Bearer " + access_token),
-            data=self.event)
-        self.assertEqual(rv.status_code, 201)
```

Comment:
you are testing for the get not the post..assert the status code in the get not the post